api/v1/policy_types.go

```diff
@@ -24,25 +24,46 @@ const (
 // PolicyTemplate template for custom security policy
 type PolicyTemplate struct {
 	// +kubebuilder:pruning:PreserveUnknownFields
-	ObjectDefinition runtime.RawExtension `json:"objectDefinition"`
+	ObjectDefinition  runtime.RawExtension `json:"objectDefinition"`
+	ExtraDependencies []PolicyDependency   `json:"extraDependencies,omitempty"`
+	IgnorePending     bool                 `json:"ignorePending,omitempty"`
```

Comment:
Could you document this field so it shows up in the CRD?

Reply:
Thanks for catching that! I also added some more comments for other fields, somewhat based on the doc at https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html/governance/governance#policy-yaml-structure